README.md

```diff
@@ -1,29 +1,28 @@
-# IBM Cloud Satellite Module
+# IBM Cloud Satellite module
 
-Use this terrafrom automation to set up satellite location on IBM cloud.
-It will provision satellite location and create 6 VSIs and assign 3 host to control plane, and provision ROKS satellite cluster and auto assign 3 host to cluster,
-Configure cluster worker pool to an existing ROKS satellite cluster.
+Use this Terraform automation to set up a Satellite location on IBM cloud. The module provisions the IBM Cloud® Satellite location, creates 6 VSIs, assigns three hosts to the control plane, provisions a ROKS Satellite cluster, assigns three hosts to the cluster, and configures a cluster worker pool to an existing ROKS Satellite cluster.
 
-This is a collection of sub modules that make it easier to provision a satellite on IBM Cloud.
-* location
-* host
-* cluster
-* configure-cluster-worker-pool
+This module is a collection of submodules that make it easier to provision a Satellite on IBM Cloud.
```

Comment:
"provision a Satellite" doesn't really make sense.  We could say "provision Satellite resources" to cover the fact it does a location, hosts, cluster, etc